packages/docusaurus-theme-classic/src/theme/Navbar/MobileSidebar/SecondaryMenu/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {type ComponentProps} from 'react';
+import {useNavbarSecondaryMenu, useThemeConfig} from '@docusaurus/theme-common';
+import type {Props as NavbarItemConfig} from '@theme/NavbarItem';
+import Translate from '@docusaurus/Translate';
+
+function useIsPrimaryMenuEmpty() {
+  // TODO temporary casting until ThemeConfig type is improved
+  return (useThemeConfig().navbar.items as NavbarItemConfig[]).length === 0;
```

Comment:
This casting shouldn't be necessary?

Reply:
`.navbar.items` is an array, no? We don't need to cast it to get `.length`?

Reply:
🤦 right 😄 updated